src/lib/clk.c

```diff
@@ -63,6 +65,9 @@ void clock_set_freq(int clock, uint32_t hz)
 					 hz);
 	clk_notify_data.freq = clk_info->freqs[idx].freq;
 
+	tr_info(&power_tr, "clock %d set freq %dkHz, freq_idx= %d",
```

Comment:
'=' is used inconsistently in the log message, pls consider removing it and just keep a single space between the name and value.

Reply:
updated